src/views/certifications/views/landing/components/exam-card/exam-card.module.css

```diff
@@ -0,0 +1,21 @@
+/* Sets padding for both card types, which differs from default card padding */
+.cardBase {
+	padding: 24px;
+}
+
+.examCard {
+	composes: cardBase;
+}
+
+.contents {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	flex-wrap: wrap;
```

Comment:
<img width="905" alt="Screen Shot 2022-12-14 at 3 04 58 PM" src="https://user-images.githubusercontent.com/36613477/207735208-3d5a0a39-a3fb-4b5c-b7bd-c25f8b017a10.png">

At some in between tablet / mobile sizes we get some odd wrapping. Not the end of the world but perhaps we want to define a column orientation for mobile up to tablet? so everything breaks evenly?

Reply:
Great call - I switched this up so we have `flex-direction: column` on mobile. Everything should break evenly to `flex-direction: row` at our custom `--dev-dot-tablet-up` breakpoint ✅